models/schema/workloads/tfgrid_workloads_reservation_1.toml

```diff
@@ -31,7 +31,7 @@ zdbs = (LO) !tfgrid.workloads.reservation.zdb.1
 networks = (LO) !tfgrid.workloads.reservation.network.1
 kubernetes = (LO) !tfgrid.workloads.reservation.k8s.1
 proxies = (LO) !tfgrid.workloads.reservation.gateway.proxy.1
-reserve_proxies = (LO) !tfgrid.workloads.reservation.gateway.reserve_proxy.1
+reserve_proxies = (LO) !tfgrid.workloads.reservation.gateway.reverse_proxy.1
```

Comment:
```suggestion
reverse_proxies = (LO) !tfgrid.workloads.reservation.gateway.reverse_proxy.1
```

Reply:
oh wow, then my vscode is broken.. Because I searched on it and didn't find any other occurences. Time to double check with the command line